src/messages/pt.json

```diff
@@ -77,7 +79,6 @@
   "api_reference_page.subtitle": "Use our API reference documentation to build custom solutions that fit your business.",
   "start_here_page.title": "Comece aqui",
   "start_here_page.subtitle": "Aprenda, passo a passo, tudo que precisa saber para começar a usar a VTEX.",
-  "start_here_page.link": "Ver tutoriais",
```

Comment:
This string was removed from all files, but it is used in the tracks landing page (`src/pages/docs/tracks`).

Reply:
I think I get it now, you changed it to `tutorials_page.link` because of the string's value, but actually it was misnamed. I guess the strings' names played tricks on us 😅

Reply:
Thanks for the review @PedroAntunesCosta 
Yes, exactly 😅 that's why the strings related to links in the page tutorials were broken.
I believe now it's fixed.